src/main/java/com/myhexin/cragent/coderabbittest/UserManager.java

```diff
@@ -0,0 +1,23 @@
+package com.myhexin.cragent.coderabbittest;
+
+import com.alibaba.fastjson.JSON;
+
+import java.util.Optional;
+
+public class UserManager {
+
+    /**
+     * 从客户信息 json 中获取身份证号
+     * {
+     *     "userInfo": {
+     *         "idCard": "xxx"
+     *     }
+     * }
+     *
+     * @param json 可能为 {@code null}
+     * @return 身份证号
+     */
+    private String getIdCard(String json) {
+        return Optional.ofNullable(json).map(JSON::parseObject).map(o -> o.getJSONObject("userInfo")).map(o -> o.getString("idCard")).orElse(null);
```

Comment:
**Issue**: The use of fastjson library can pose security risks if not properly handled, especially with untrusted input. <br> **Fix**: Consider using a more secure JSON parsing library or ensure that the input is validated and sanitized before parsing. <br> **Code Suggestion**: 
 ```
 - import com.alibaba.fastjson.JSON;
 + import com.fasterxml.jackson.databind.ObjectMapper;

 - JSON::parseObject
 + new ObjectMapper().readValue(json, JSONObject.class)
 ```